components/n-ui/tracking/ft/index.js

```diff
@@ -104,9 +106,17 @@ const oTrackingWrapper = {
 				useSendBeacon: flags.get('sendBeacon')
 			});
 
-			//headline testing
-			const headlineTestingVariant = document.querySelector('[data-trackable-headline-variant]') ? document.querySelector('[data-trackable-headline-variant]').getAttribute('data-trackable-headline-variant') : null;
-			context['headline-variant'] = headlineTestingVariant;
+			//headline testing, add variant to the page view event as long as there is only one article under test
+			if (location.pathname === '/') {
+				const alternativeHeadlines = document.querySelectorAll('[data-trackable-headline-variant]');
+				let hrefs = [];
+				alternativeHeadlines.forEach(headline => hrefs.push(headline.getAttribute('href')));
+
+				const isOnlyOneArticle = hrefs.length ? !!hrefs.reduce((a, b) => a === b ? a : NaN) : false;
```

Comment:
Maybe I've missed something but why `NaN` here?

Reply:
Good point, could have just been false.
Have updated it to slice the NodeList as you suggested, and changed the href equality check to use Array.every() as it's a bit neater.